Tiltfile

```diff
@@ -383,3 +395,14 @@ if localnet_config["rest"]["enabled"]:
     print("REST enabled: " + str(localnet_config["rest"]["enabled"]))
     deployment_create("rest", image="davarski/go-rest-api-demo")
     k8s_resource("rest", labels=["data_nodes"], port_forwards=["10000"])
+
+### Pocketdex Shannon Indexer
+load("./tiltfiles/pocketdex.tilt", "check_and_load_pocketdex")
```

Comment:
For context, I just reviewed the comments in the other .tilt files (not the code), but lgtm